cmd/containerd-shim-runhcs-v1/task_hcs.go

```diff
@@ -147,12 +147,11 @@ func newHcsTask(
 	}
 
 	opts := hcsoci.CreateOptions{
-		ID:                      req.ID,
-		Owner:                   owner,
-		Spec:                    s,
-		HostingSystem:           parent,
-		NetworkNamespace:        netNS,
-		ScaleCPULimitsToSandbox: shimOpts.ScaleCpuLimitsToSandbox,
```

Comment:
Why did we remove this assignment?

Reply:
I was very confused, because I thought we already fixed this in #913... Turns out the issue was re-introduced, probably due to a bad merge, [here](https://github.com/microsoft/hcsshim/commit/88e182c049ac7b5a99e72d778248b518e765f8e9#diff-04078e400c5f445fb34c4927b97930a0d1dbd00005819d7b7a77022d092f365c). :(

Reply:
I cry

Reply:
looks like it was noticed just before / after merge 🙈 https://github.com/microsoft/hcsshim/pull/915#discussion_r587693490

Reply:
I thought I force pushed to fix before checking in 😭 😭 😭. Well I've gotta give it to myself, I both fixed it and then broke it again. This deserves a medal

Reply:
💩 happens; you did that to test if the linters work, correct? 😂

Reply:
100%, don't let anyone tell you otherwise 😉